ansible/roles/ocp4-workload-oadp/tasks/post_workload.yml

```
@@ -61,6 +61,7 @@
         apps_url: "{{ apps_url | default('appsurl') }}"
       # yamllint disable-line rule:line-length
         cluster_admin_user: "{{ cluster_admin_user | default(ocp4_workload_oadp_cluster_admin) | default('kubeadmin') }}"
+        cluster_admin_password: "{{ cluster_admin_password | default(ocp4_workload_oadp_cluster_admin_password) | default('undefined') }}"
         api_url: "{{ api_url_results.stdout }}"
         ocp4_password: "{{ student_password | default('undefined')}}"
```

```suggestion
        cluster_admin_password: "{{ cluster_admin_password | default(ocp4_workload_oadp_cluster_admin_password) | default('undefined') }}"
        api_url: "{{ api_url_results.stdout }}"
```

if you prefer to keep the var names accurate to agnosticd in bookbag then perhaps just remove ocp4_password here entirely. this means you must change the var in the bookbag also.. alternatively stuff cluster_admin_password back into ocp4_password.

---

I think cluster-admin_password is less confusing than ocp4_password. Also, MTC is using ocp4_password = student password, so let's add new variable instead changing ocp4_password. We might need 'student password' in future.

---

@hhpatel14 click resolve.